### PR TITLE
Feature UI edit compspec

### PR DESCRIPTION
**Pull request by dylanmartin**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our [guidelines](../CONTRIBUTING.md)
- [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
- [x] Passes e2e testing
- [x] Passes lint

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature

* **What is the current behavior?** (You can also link to an open issue here)
Compspecs can not be edited via the UI

* **What is the new behavior (if this is a feature change)?**
Compspecs can now be edited on the computation page if the user has the correct permissions

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: packages/coinstac-ui/app/render/components/computations/computation-io.jsx [hunk 1]
```diff
@@ -1,31 +1,125 @@
...
+
+    try {
+      delete compSpecDbObject.id;
+      delete compSpecDbObject.__typename;
+      delete compSpecDbObject.computation.__typename;
+      delete compSpecDbObject.computation.remote.__typename;
+      delete compSpecDbObject.meta.__typename;
+      delete compSpecDbObject.submittedBy;
+    } catch (e) {
+      console.error(e);
```
Comment (rssk) [inline packages/coinstac-ui/app/render/components/computations/computation-io.jsx]: Add a logger call here instead:-)

File: packages/coinstac-ui/app/render/state/apollo-client.js [hunk 2]
```diff
@@ -69,6 +69,7 @@ function getApolloClient({ apiServer, subApiServer }) {
     client: new ApolloClient({
       link: unauthorizedLink.concat(splitLink),
       cache: new InMemoryCache({
+        // addTypename: false,
```
Comment (rssk) [inline packages/coinstac-ui/app/render/state/apollo-client.js]: remove comment

File: packages/coinstac-headless-client/src/apollo-client.js [hunk 3]
```diff
@@ -41,7 +41,7 @@ function createApolloClient(authToken) {
 
   return new ApolloClient({
     link: concat(authMiddleware, splitLink),
-    cache: new InMemoryCache(),
+    cache: new InMemoryCache({ addTypename: false }),
```
Comment (rochaeb) [inline packages/coinstac-headless-client/src/apollo-client.js]: I think it's better to remove this, since it was giving problems in the ui

**Review by rochaeb — APPROVED** [review]
Just made one comment. Besides that it's looking good.

**Pull request merged**